### PR TITLE
Sprint 7/1

### PR DESCRIPTION
**Pull request by kasiaekiert**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
```
Comment (piowit) [inline config/environments/development.rb:36]:
```suggestion
```
lets remove unused line, as `:local` line is overwritten in the next line

**Review by piowit — COMMENTED** [review]
Looks good 🔥 💪🏻
I put below just one suggestion.

**Review by ciejj — APPROVED** [review]
👍

**Comment by ciejj** [comment]
Agree on commit messages  with @piowit  :)

**Pull request merged**